pycqed/instrument_drivers/meta_instrument/qubit_objects/QuDev_transmon.py

```diff
@@ -2519,20 +2491,19 @@ def find_RB_gate_fidelity(self, cliffords, nr_seeds, label=None,
 
         #Analysis
         if analyze:
-            pulse_length = self.gauss_sigma() * self.nr_sigma()
             if interleaved_gate is None:
-                rbma.RandomizedBenchmarking_Analysis(label=label,
-                                 qb_name=self.name,
-                                 T1=T1, T2=T2, pulse_length=pulse_length,
-                                 gate_decomp=gate_decomposition)
+                if classified_ro:
+                    ba3.BaseDataAnalysis()
+                else:
+                    print('Currently only classified ro is supported.')
```

Comment:
`print` -> `log.error`

Reply:
Additional support has been added in Proj_XLD.

Reply:
will not change then